push/service_test.go

```diff
@@ -83,7 +83,7 @@ func TestTimestampError(t *testing.T) {
 
 	handler.HandleFunc("/3/device/", func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusGone)
-		w.Write([]byte(`{"reason":"Unregistered","timestamp":12622780800}`))
+		w.Write([]byte(`{"reason":"Unregistered","timestamp":1262278080000}`))
```

Comment:
I get http://play.golang.org/p/ctPrCvcsWV
2370-01-01 00:00:00 +0000 UTC
2009-12-31 16:48:00 +0000 UTC

But I guess it doesn't matter either way, as this is just the test.